website/docs/r/app_service_connection.html.markdown

```diff
@@ -88,7 +88,7 @@ The following arguments are supported:
 
 * `target_resource_id` - (Required) The ID of the target resource. Changing this forces a new resource to be created. Possible values are `Postgres`, `PostgresFlexible`, `Mysql`, `Sql`, `Redis`, `RedisEnterprise`, `CosmosCassandra`, `CosmosGremlin`, `CosmosMongo`, `CosmosSql`, `CosmosTable`, `StorageBlob`, `StorageQueue`, `StorageFile`, `StorageTable`, `AppConfig`, `EventHub`, `ServiceBus`, `SignalR`, `WebPubSub`, `ConfluentKafka`.
 
-* `authentication` - (Required) The authentication info. An `authentication` block as defined below.
+* `authentication` - (Required) The authentication info.  An `authentication` block as defined below.
```

Comment:
I don't think all the comments have been resolved yet, according to https://github.com/hashicorp/terraform-provider-azurerm/pull/20643#discussion_r1142216745

There should be a Note below this field, also there's an extraneous space here 
```suggestion
* `authentication` - (Required) The authentication info. An `authentication` block as defined below.
```

Reply:
Hi @stephybun , sorry for missing the `note` below. Just fixed this.